_extensions/bluesky-comments/bluesky-comments.lua

```diff
@@ -38,11 +36,17 @@ local function getFilterConfig(meta)
     filterConfig.filterEmptyReplies = config['filter-empty-replies']
   end
 
+  if config['n-show-init'] then
+    filterConfig.nShowInit = tonumber(pandoc.utils.stringify(config['n-show-init']))
+  end
+
   if config['visible-comments'] then
+    utils.log_warn("`visible-comments` was deprecated, please use `n-show-init` instead.")
```

Comment:
Should this be `n-show-more`? Right now, that option doesn't seem to be getting exported over to the JS Library?

Reply:
I was relying on some logic in `getFilterConfig()` that would take additional metadata attributes and pass them through the `config` object. I'd rather use attributes on the custom element instead, so I've removed that logic and now additional `kwargs` are passed to the custom element directly, such that `{{< bluesky-comments n-show-init="3" >}}` becomes `<bluesky-comments n-show-init="3">`.

This makes the web component more portable and it's a cleaner setup for the JSON blob which is now used only for `filterConfig` (or the `filter-config` attribute in the custom element).

This happened in c1cfed12f2fbf4fbe4f52e813dea22f2e7f77af1.

Reply:
Actually, this ended up need more work than that linked commit, but now we handle inline attributes equivalently to the `bluesky-comments` metadata, taking the inline attributes over the metadata values (or in the case of `mute-patterns` and `mute-users` we merge inlines with metadata).